class UnitExistsError(Exception):
    pass


class UnitNotExistsError(Exception):
    pass
